e2e-tests/demand-backup-sharded/conf/some-name-rs0.yml

```diff
@@ -34,6 +34,14 @@ spec:
           bucket: operator-testing
           prefix: psmdb
           endpointUrl: https://storage.googleapis.com
+      azure-blob:
+        type: azure
+        azure:
+          account: pxcperator
```

Comment:
```suggestion
          account: k8soperators
```

Reply:
Resolved in e23e34aa6fe192bf81d1378f5c0e3fb76ec60d85